app/models/labels/plate_label_human_barcode.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class Labels::PlateLabelHumanBarcode < Labels::Base # rubocop:todo Style/Documentation
```

Comment:
It does not make much difference but can `Labels::PlateLabelHumanBarcode` be the subclass of `Labels::PlateLabel` or `Labels::PlateLabelBase` instead of `Labels::Base` ?